src/apps/something-similar/SomethingSimilarList.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, FC, useState } from "react";
+import {
+  useRecommendFromFaustQuery,
+  RecommendFromFaustQuery,
+  useSearchWithPaginationQuery,
+  SearchWithPaginationQuery
+} from "../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../core/utils/text";
+import { Work } from "../../core/utils/types/entities";
+import fetchMaterial, {
+  MaterialProps
+} from "../loan-list/materials/utils/material-fetch-hoc";
+import RecommendMaterial from "../recommender/RecommendMaterial";
+
+export interface LikeThisListProps {
+  id: string;
+}
+
+const LikeThisList: FC<LikeThisListProps & MaterialProps> = ({
+  id,
+  material
+}) => {
+  const t = useText();
+  const [recommendView, setRecommendView] = useState<boolean>(true);
+  const { data: somethingSimilarData } = useRecommendFromFaustQuery({
+    faust: id,
+    limit: 4
+  });
+
+  const { data: byAuthorData } = useSearchWithPaginationQuery({
+    limit: 4,
+    q: {
+      all: material?.firstAuthor
+    },
+    offset: 0
+  });
+
+  const [somethingSimilar, setSomethingSimilar] =
```

Comment:
It is more of a cosmetic thing, but I think it would be nice to group the useState hooks in the top.